include/libtorrent/torrent.hpp

```diff
@@ -53,40 +53,45 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/logic/tribool.hpp>
 #include "libtorrent/aux_/disable_warnings_pop.hpp"
 
-#include "libtorrent/fwd.hpp"
```

Comment:
what happened to all the include directives?

Reply:
They got reordered by the linter... I'll disable it and put them back as before.